extensions/pkg/controller/worker/genericactuator/actuator_delete.go

```diff
@@ -122,10 +127,16 @@ func (a *genericActuator) Delete(ctx context.Context, worker *extensionsv1alpha1
 	}
 
 	// Cleanup machine dependencies.
+	// TODO(dkistner) DEPRECATED: Remove in a future release.
```

Comment:
The corresponding func should be already marked as deprecated, I see no reason to say in each invocation of the func that it is deprecated.

```suggestion
	// TODO(dkistner): Remove in a future release.
```

Reply:
Well does it matter too much? Just another reminder that it is deprecated and need to be cleaned up in the future. Removed the deprecation notes in the reconcile and delete flow and just kept them on the interface.